mappings/net/minecraft/entity/mob/Angerable.mapping

```diff
@@ -17,6 +20,10 @@ CLASS net/minecraft/class_5354 net/minecraft/entity/mob/Angerable
 		ARG 1 player
 	METHOD method_29517 angerToTag (Lnet/minecraft/class_2487;)V
 		ARG 1 tag
+	METHOD method_29921 universallyAnger ()V
```

Comment:
```suggestion
	METHOD method_29921 setUniversalAnger ()V
```

Reply:
Fyi `anger` is already a verb and is perfectly suitable as a method name.